as/tests/org.jboss.ide.eclipse.as.management.as7.tests/src/org/jboss/ide/eclipse/as/internal/management/as7/tests/utils/StartupUtility.java

```diff
@@ -81,6 +78,27 @@ public static Process runServer(String homeDir) {
 		System.out.println("Somehow launch completed without returning a process");
 		return null;
 	}
+	
+	private static String getJavaHome(String rtType, String serverHome) {
```

Comment:
suggestion: call it checkAndGetJavaHome() since its actually not just getting the javahome.